splipy/curve_factory.py

```diff
@@ -191,38 +191,6 @@ def ellipse(r1=1, r2=1, center=(0,0,0), normal=(0,0,1), type='p2C0', xaxis=(1,0,
     result.rotate(rotate_local_x_axis(xaxis, normal))
     return flip_and_move_plane_geometry(result, center, normal)
 
-def ellipse(r1=1, r2=1, center=(0,0,0), normal=(0,0,1), type='p2C0', xaxis=(1,0,0)):
```

Comment:
Fixed ellipse factory by deleting it? Or am I missing something?

Reply:
There were two different methods with the same name and signature and no unit tests.

Reply:
Okay :+1: